media-libs/libfpx/libfpx-1.3.1_p10-r1.ebuild

```diff
@@ -18,6 +18,7 @@ IUSE="static-libs"
 PATCHES=(
 	"${FILESDIR}"/${PN}-1.2.0.13-export-symbols.patch
 	"${FILESDIR}"/${PN}-1.3.1_p10-musl-1.2.3-null.patch
+	"${FILESDIR}"/${PN}-1.3.1_p10-remove-register-keyword.patch
```

Comment:
I think it's better to add `-std=gnu++17` or `-std=gnu++14` temporarily with a comment in the ebuild that it can be dropped on the next release (with a link to the PR/bug).